frontend/packages/operator-lifecycle-manager/src/types.ts

```diff
@@ -53,6 +53,12 @@ export enum SubscriptionState {
   SubscriptionStateAtLatest = 'AtLatestKnown',
 }
 
+export enum ClusterServiceVersionStatus {
+  ClusterServiceVersionStatusFailed = 'Failed',
```

Comment:
Nit: why the `ClusterServiceVersionStatus` prefix for all enum members?

Reply:
the other enums in this file/package use the same naming so I went with what is already used to stay consistent